apps/web/src/pages/blocks.tsx

```diff
@@ -218,7 +218,7 @@ const Blocks: NextPage = function () {
       paginationData={{ pageSize: ps, page: p }}
       isExpandable
       rowSkeletonHeight={44}
-      tableTopSlot={<Filters />}
+      tableTopSlot={<Filters enableSlotFilter />}
```

Comment:
Is this only enabled for blocks on purpose?

Reply:
Yes! You can check the ticket info :+1: